### ISSUE TITLE
Add a closing animation more friendly

### ISSUE THREAD

**Issue by Richard38907** [FossifyOrg/Gallery#250]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Gallery/issues) or [closed](https://github.com/FossifyOrg/Gallery/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Gallery/discussions) or [closed](https://github.com/FossifyOrg/Gallery/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Gallery/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Add an animation when closing photos like this

https://github.com/FossifyOrg/Gallery/assets/148692265/e9957df4-b222-40f4-97f7-417f6cce5c7b

### Why do you want this feature?

More friendly and smooth

**Comment by Richard38907** [comment]
The app I use in the video is Google Photos

**Comment by thisisartman** [comment]
I agree. It's all about the UX. Swiping down and seeing the photo going down with the photo grid visible is a definite plus point. Helps people understand what a gesture is gonna do.
